Framework/Crystal/src/IndexPeaks.cpp

```diff
@@ -382,6 +385,9 @@ void IndexPeaks::init() {
   this->declareProperty(Prop::SAVEMODINFO, false,
                         "If true, update the OrientedLattice with the maxOrder, "
                         "modulation vectors & cross terms values input to the algorithm");
+  this->declareProperty(Prop::UPDATE_UB, false,
```

Comment:
This option only applies to peak tables with peaks from 1 run and when `CommonUBForAll=False` - I wonder whether it would be possible to only enable the option when `CommonUBForAll=False` and add some input validation method to check the peak table includes peaks form only 1 run? Or at least throw a warning if it has peaks from more than 1 run saying the option `UpdateUB` will be ignored.

Reply:
I am not sure if it is possible to enable a property based on another one because the properties are defined when you initialise the algorithm, but I will check that. If it is not possible, then I will add a warning